xarray/plot/utils.py

```diff
@@ -360,15 +360,37 @@ def _infer_xy_labels(darray, x, y, imshow=False, rgb=None):
             raise ValueError("DataArray must be 2d")
         y, x = darray.dims
     elif x is None:
-        if y not in darray.dims and y not in darray.coords:
+        if (
+            y not in darray.dims
+            and y not in darray.coords
+            and y not in darray._level_coords
+        ):
             raise ValueError("y must be a dimension name if x is not supplied")
```

Comment:
```suggestion
            raise ValueError("y must be None, a dimension name, coordinate variable or coordinate level")
```

 1. Better suggestions would be welcome.
 2. I think the second part (`x is not supplied`) is not necessary - can anyone confirm?